www/pages/meeting/[id]/index.js

```diff
@@ -67,14 +67,15 @@ const Meeting = ( props ) => {
   useEffect( () => {
     const save = async() => {
       const meeting_id = router.query.id || '';
+      const real_id = meeting_id.length === 24;
 
       try {
         await client.post(
           'meeting/aggregate',
           {
             name,
             owner_id: user._id,
-            meeting_id,
+            ...( real_id && { meeting_id } ),
```

Comment:
This is dope, usually I do something like:

```
const should_have_date = true;

const meeting = {
  name: 'something',
  date: should_have_date ? new Date() : undefined
};
```
but the the `date` key is still defined. The problem with that, is that mongo still adds the key to documents when inserting meaning there is some wasted space.